client/stylesheets/_toCat.scss

```diff
@@ -3645,13 +3645,13 @@ On click the images will be displayed at normal size to complete the effect
 .profileUserRight {
     width: 55%;
     vertical-align: top;
-    margin-top: 1.2rem;
+    margin-top: 0rem !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>`0rem` should be written without units as `0`